batch/src/main/resources/log4j2.xml

```diff
@@ -1,29 +1,29 @@
-<?xml version="1.0" encoding="UTF-8" ?>
+<?xml version="1.0" encoding="UTF-8"?>
 <!--
- Copyright 2020 Google LLC
-
- Licensed under the Apache License, Version 2.0 (the "License");
- you may not use this file except in compliance with the License.
- You may obtain a copy of the License at
-
-      http://www.apache.org/licenses/LICENSE-2.0
-
- Unless required by applicable law or agreed to in writing, software
- distributed under the License is distributed on an "AS IS" BASIS,
- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- See the License for the specific language governing permissions and
- limitations under the License.
+//Copyright 2020 Google LLC
```

Comment:
Why are these C/Java style comment symbols added to an XML file?

Reply:
@bashir2  from https://github.com/GoogleCloudPlatform/openmrs-fhir-analytics/pull/87#pullrequestreview-562828947 we can support tab to indent with `<tabIndent>` but `<indentSize> ` will be ignored as advised [here ](https://acegi.github.io/xml-format-maven-plugin/xml-format-mojo.html#tabIndent)under the available configuration options for the xml formater

Reply:
> @bashir2 from [#87 (review)](https://github.com/GoogleCloudPlatform/openmrs-fhir-analytics/pull/87#pullrequestreview-562828947) we can support tab to indent with `<tabIndent>` but `<indentSize> ` will be ignored as advised [here ](https://acegi.github.io/xml-format-maven-plugin/xml-format-mojo.html#tabIndent)under the available configuration options for the xml formater

I am not saying that we should use tab for indent, in fact I prefer space. What I am questioning is that  why this plugin has _not_ fixed the current mix, i.e., why it has not converted the tabs in `pom.xml` to spaces?